src/pages/index.js

```diff
@@ -119,3 +243,9 @@ const editFormValidator = new FormValidator(
   profileEditForm
 );
 editFormValidator.enableValidation();
+
+// const editAvatarValidator = new FormValidator(
+//   formValidatorConfig,
+//   avatarSubmitButton
```

Comment:
`avatarSubmitButton`


You need to pass the `avatar form` into `FormValidator` rather than the button